otx/mpa/det/incremental/trainer.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+import glob
+import os.path as osp
```

Comment:
Do we need all of this imports if we create only one class without any new methods/attributes?

Reply:
I missed that. Thanks. I removed them